lib/gyp/generator/ninja/index.js

```diff
@@ -350,8 +353,8 @@ Ninja.prototype.actionCmd = function actionCmd(base, toBase, cmds) {
   if (this.flavor !== 'win32')
     return res;
 
-  // TODO(indutny): escape quotes in res
-  return `cmd.exe /s /c "${res}"`;
+  const nw = gyp.platform.win.getNinjaWrapper(this.targetArch);
+  return `${nw} cmd.exe /s /c "${res}"`;
```

Comment:
`res = '"gaga"'` =>  `${nw} cmd.exe /s /c ""gaga""`  and that's fine or even `${nw} cmd.exe /s /c ""gaga" foo "bar""` and that's fine too `/s` is magic